server/routes/model.routes.js

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import { authenticate, isAdminOrUserOrVendor } from '../helpers/middleware';
+import ModelController from '../controllers/model.controllers';
+
+const router = express.Router();
+
+/**
+ * @swagger
+ * path:
+ *  /:
+ *    get:
+ *      summary: Returns all models in database
+ *      tags: [Models]
+ *      responses:
+ *        "200":
+ *          description: All models
+ */
+router.get('/', authenticate, isAdminOrUserOrVendor, ModelController.getAllModels);
```

Comment:
Using `Model` gives me a strange meaning (not sure if it is just me or it would make understanding the purpose of the model unclear. Lets change this to something totally different like TotalCount. 

Kindly fix the filenames and other related stuff too 🙏🏽 

```suggestion
router.get('/', authenticate, isAdminOrUserOrVendor, TotalCountController.getAllModels);
```